Abstract.md

```diff
@@ -1,38 +1,40 @@
 # Abstract for conferences
 
-Title proposal: 
+Title proposal:
 
-End-to-end workflow for LC-MS/MS pre-processing and analysis using 
-open-source *xcms* software and the *RforMassSpectrometry* environment.
+End-to-end workflow for reproducible analysis of LC-MS/MS data in R using
+*xcms* and packages from the *RforMassSpectrometry* initiative.
 
-OR 
+OR
 
-Streamlining LC-MS/MS Data Analysis with Open-Source *xcms* and 
+Streamlining LC-MS/MS Data Analysis in R with Open-Source *xcms* and
```

Comment:
I like this title, but I would specify that this is a workflow in R - there might be others in Python or others that can be done with Mzmine etc